packages/expect-utils/src/__tests__/isError.test.ts

```diff
@@ -46,16 +46,19 @@ describe('isError', () => {
   });
 
   it('should detect errors from another context', () => {
-    testErrorFromDifferentContext(win => new win.Error());
+    testErrorFromDifferentContext(
+      win => new (win as typeof globalThis).Error(),
```

Comment:
`Error` does not exist on type `Window`, but with `typeof globalThis`. Is fine? Or?